cmd/ctr/start.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"github.com/containerd/console"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var startCommand = cli.Command{
+	Name:      "start",
+	Usage:     "start a container that have been created",
```

Comment:
Ya, that is fine, just keep in mind, `ctr` is a dev tool, its not to be supported.  It's just a way to exercise the API and test things that are implemented in the long term.